Formula/depqbf.rb

```diff
@@ -32,7 +32,12 @@ def install
     (buildpath/"picosat-960").install resource("picosat")
     system "./compile.sh"
     bin.install "depqbf"
-    lib.install "libqdpll.a", "libqdpll.1.0.dylib"
+    on_macos do
+      lib.install "libqdpll.a", "libqdpll.1.0.dylib"
+    end
+    on_linux do
+      lib.install "libqdpll.a", "libqdpll.so.1.0"
+    end
```

Comment:
Since the first one isn't specific I think we should leave it out of the conditional block
```suggestion
    lib.install "libqdpll.a"
    on_macos do
      lib.install "libqdpll.1.0.dylib"
    end
    on_linux do
      lib.install "libqdpll.so.1.0"
    end
```